src/schema/fair.ts

```diff
@@ -155,8 +113,8 @@ const FairType = new GraphQLObjectType({
       type: Profile.type,
       resolve: (
         { default_profile_id, organizer },
-        options,
-        request,
+        _options,
```

Comment:
Why prefix with underscores?

Reply:
There's sort of a convention (not specific to any language or framework), that variables which are unused get prefixed with an underscore (for reader clarity).

Reply:
It's a way to silence `jslint` when a variable is unused

Reply:
Also, there are languages where you can't ignore any params, and the underscore is meant to mean intentionally left blank